### DISCUSSION TITLE
How to create shared library out of this

### DISCUSSION THREAD

**Discussion by prasyadav** [ArthurSonzogni/FTXUI#545] [Q&A] [answered]
Hello,
I wanted to create a shared library of ftxui api to use in my project.
how can i do that?

**Comment by ArthurSonzogni** [comment, marked as answer]
Hello!
Do you really need "shared libraried"? You can download the static libraries from the release page.
Alternatively, and this is the recommended way, you can use cmake to build yourself the latest version. You can even tell build shared libraries if you want. This is already explained in the Readme how to build. This is standard C++/CMake.

You can also try building the ftxui-starter project.
